frame/support/src/traits/tokens/nonfungibles_v2.rs

```diff
@@ -180,6 +180,16 @@ pub trait InspectEnumerable<AccountId>: Inspect<AccountId> {
 	) -> Self::OwnedInCollectionIterator;
 }
 
+/// Trait for providing an interface to read-only the role of NFT-like.
+pub trait InspectRole<AccountId>: Inspect<AccountId> {
+	// Return `true` if collection issuer.
```

Comment:
1) pls use `///` instead of `//` here and below
2) I would copy the description from the above ("Returns `true` if `who` is the issuer of the `collection`.")